### ISSUE TITLE
Company is confusing on signup

### ISSUE THREAD

**Issue by G-Bro** [Teemill/teemill#46]
on signup form, company could be misinterpreted as the company or brand you are creating with your site, i.e. the website url or the h1 on your store

**Comment by martdog** [comment]
Are they not the same thing? I.e. Squigglers = Squigglers?

**Comment by G-Bro** [comment]
This is an issue with the login form where the user puts in company/organisation
![image](https://cloud.githubusercontent.com/assets/17454235/13328088/af369f46-dbe4-11e5-88a7-5602c440dea2.png)

**Comment by martdog** [comment]
Makes sense. I think we could use it positively, i.e. Prefill the "store name" with company field

**Comment by martdog** [comment]
"I dont know what to call it yet" - is an interesting issue... linked to logo generator in theme. But IMO Teemill a product for those with a ready plan.

**Comment by martdog** [comment]
![download](https://cloud.githubusercontent.com/assets/17437229/13328173/2ab04230-dbe5-11e5-8e4d-0cbdc263e560.jpg)

**Comment by G-Bro** [comment]
![image](https://cloud.githubusercontent.com/assets/17454235/13328194/4da3972e-dbe5-11e5-91b6-d6ea7fa525a9.png)

**Comment by martdog** [comment]
Tooltip "dont know what to call your brand?" to this blog (_blank)

https://teemill.co.uk/blog/5-steps-to-a-winning-brand-name/
